### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by 9joshua** [osTicket/osTicket#6777] [General]
I noticed the last commit to the dev branch was 14 months ago, so curious if osTicket is more or less abandoned.

This is really important to know as users will soon need to find a replacement solution without future PHP deprecation fixes and security patches.

**Comment by JediKev** [comment]
@9joshua 

I need you to go search this topic on the forum and on here and go read the endless posts about it. No it’s not dead. Look at commit history. Look at recent pulls. Look at recent posts on forum. There will be new releases coming soon. Current legacy codebase is in maintenance mode whilst we focus on v2.0. Doesn’t mean it’s dead just means release cycles have slowed down to only important updates.

Cheers.
